api.raml

```diff
@@ -656,3 +789,72 @@ schemas:
                     }
                   ]
 # TODO: need docs for new search api, i.e., /user?username=AAA&email=AAA@example.org
+/mfa:
+  post:
+    description: Create a new mfa record.
+    body:
+      application/json:
+        schema: MfaCreation
+        example: |
+          {
+            "employee_id": "12345",
+            "type": "backupcode"
+          }
+    responses:
+      200:
+        description: The new mfa record was created.
+        body:
+          application/json:
+            schema: MfaCreationResponse
+            example: |
+              {
+                "id": 3,
+                "data": [
```

Comment:
data is passed through from the backend mostly, so it could be different, but in this case we can also make it an object with a `codes` attribute that is an array of values. @wcjr do you care from the UI if data is an array or an object and you access the `.codes` attribute?